Build failing on gpm_imerg_early_chunk_time

In #48, @davidbrochart udpated the gpm_imerg_early dataset to include a new chunking scheme. The tests pasts on the PR, but now the master build is failing

https://travis-ci.org/pangeo-data/pangeo-datastore/builds/584607961

```python
=================================== FAILURES ===================================
______ test_intake_dataset_to_dask[atmosphere.gpm_imerg_early_chunk_time] ______
catalog = <Intake catalog: master>
dataset_name = 'atmosphere.gpm_imerg_early_chunk_time'
    def test_intake_dataset_to_dask(catalog, dataset_name):
        item = catalog[dataset_name]
        try:
>           ds = item.to_dask()
tests/validate_datasets.py:33: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/srv/conda/lib/python3.6/site-packages/intake_xarray/base.py:70: in to_dask
    return self.read_chunked()
/srv/conda/lib/python3.6/site-packages/intake_xarray/base.py:45: in read_chunked
    self._load_metadata()
/srv/conda/lib/python3.6/site-packages/intake/source/base.py:117: in _load_metadata
    self._schema = self._get_schema()
/srv/conda/lib/python3.6/site-packages/intake_xarray/base.py:19: in _get_schema
    self._open_dataset()
/srv/conda/lib/python3.6/site-packages/intake_xarray/xzarr.py:35: in _open_dataset
    self._ds = xr.open_zarr(self._mapper, **self.kwargs)
/srv/conda/lib/python3.6/site-packages/xarray/backends/zarr.py:448: in open_zarr
    ds = maybe_decode_store(zarr_store)
/srv/conda/lib/python3.6/site-packages/xarray/backends/zarr.py:436: in maybe_decode_store
    drop_variables=drop_variables)
/srv/conda/lib/python3.6/site-packages/xarray/conventions.py:480: in decode_cf
    ds = Dataset(vars, attrs=attrs)
/srv/conda/lib/python3.6/site-packages/xarray/core/dataset.py:384: in __init__
    self._set_init_vars_and_dims(data_vars, coords, compat)
/srv/conda/lib/python3.6/site-packages/xarray/core/dataset.py:406: in _set_init_vars_and_dims
    data_vars, coords, compat=compat)
/srv/conda/lib/python3.6/site-packages/xarray/core/merge.py:376: in merge_data_and_coords
    indexes=indexes)
/srv/conda/lib/python3.6/site-packages/xarray/core/merge.py:453: in merge_core
    dims = calculate_dimensions(variables)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
variables = OrderedDict([('lat', <xarray.IndexVariable 'lat' (lat: 1800)>
array([-89.95, -89.85, -89.75, ...,  89.75,  89.85,  89.....000000000',
       '2001-01-26T23:00:00.000000000', '2001-01-26T23:30:00.000000000'],
      dtype='datetime64[ns]'))])
    def calculate_dimensions(variables):
        """Calculate the dimensions corresponding to a set of variables.
    
        Returns dictionary mapping from dimension names to sizes. Raises ValueError
        if any of the dimension sizes conflict.
        """
        dims = OrderedDict()
        last_used = {}
        scalar_vars = set(k for k, v in variables.items() if not v.dims)
        for k, var in variables.items():
            for dim, size in zip(var.dims, var.shape):
                if dim in scalar_vars:
                    raise ValueError('dimension %r already exists as a scalar '
                                     'variable' % dim)
                if dim not in dims:
                    dims[dim] = size
                    last_used[dim] = k
                elif dims[dim] != size:
                    raise ValueError('conflicting sizes for dimension %r: '
                                     'length %s on %r and length %s on %r' %
>                                    (dim, size, k, dims[dim], last_used[dim]))
E                   ValueError: conflicting sizes for dimension 'time': length 11520 on 'time' and length 15060 on 'precipitationCal'
```

We need to fix this asap. For some reason the main website catalog is not even being populated.

--- a/intake-catalogs/atmosphere.yaml
+++ b/intake-catalogs/atmosphere.yaml
@@ -14,32 +14,6 @@ sources:
       urlpath: 'gcs://pangeo-data/gmet_v1.zarr'
       consolidated: True
 
-  gpm_imerg_early_chunk_time:
-    description: Early rainfall estimates from NASA's Global Precipitation Measurement mission (chunked along time)
-    metadata:
-      url: 'https://pmm.nasa.gov/data-access/downloads/gpm'
-      tags:
-        - precipitation
-        - satellite
-    driver: zarr
-    args:
-      urlpath: 'gcs://pangeo-data/gpm_imerg/early/chunk_time'
-      storage_options:
-        token: anon
-
-  gpm_imerg_early_chunk_space:
-    description: Early rainfall estimates from NASA's Global Precipitation Measurement mission (chunked along lat/lon)
-    metadata:
-      url: 'https://pmm.nasa.gov/data-access/downloads/gpm'
-      tags:
-        - precipitation
-        - satellite
-    driver: zarr
-    args:
-      urlpath: 'gcs://pangeo-data/gpm_imerg/early/chunk_space'
-      storage_options:
-        token: anon
-
   trmm_3b42rt:
     description: Near real time rainfall estimates from NASA's Tropical Rainfall Measuring Mission
     metadata:

